### PR TITLE
fix(forum): remediate an issue where long strings dont wrap

### PR DESCRIPTION
**Pull request by wescopeland**
This PR remediates an issue where a long string, such as a very long URL, does not break properly on forum posts and instead overflows the x-axis of the container.

This can be reproduced by going here and scrolling horizontally:
https://retroachievements.org/viewtopic.php?t=13871&c=154476#154476

The post that is overflowing is by a user named ExTermi.

After this PR, the same post wraps correctly:
Screenshot 2023-02-14 at 9 47 09 PM

### CODE REVIEW

File: public/viewtopic.php [hunk 1]
```diff
@@ -207,7 +207,7 @@
             echo userAvatar($nextCommentAuthor, label: false, iconSize: 64);
             echo "</td>";
 
-            echo "<td class='w-full py-3' id='$nextCommentID'>";
+            echo "<td class='w-full py-3 break-normal' id='$nextCommentID'>";
```
Comment (Jamiras) [inline public/viewtopic.php:210]: This doesn't seem to have any effect in Chrome or Edge. It does seem to work in Firefox.

`break-all` does appear to work in all three, but I don't now the implications of using it, or if there's a better middle ground.

Comment (wescopeland) [reply public/viewtopic.php:210]: `break-all` should be fine, although I am seeing `break-normal` work on Chrome and Edge. Is it possible on your local machine that Tailwind isn't picking up & adding the class name to the compiled CSS?

Comment (Jamiras) [reply public/viewtopic.php:210]: Anything is possible, but running `npm run build` doesn't have any noticeable effect when using `break-normal`, but was required when using `break-all`.

Comment (Jamiras) [reply public/viewtopic.php:210]: And to echo my previous statement, it did work in Firefox, so that seems to have picked it up.

Comment (wescopeland) [reply public/viewtopic.php]: Confirmed this was an issue with Tailwind & Vite on my local setup. Fixed.

**Review by Jamiras — APPROVED** [review]

**Pull request merged**